recipe/bld.bat

```diff
@@ -12,6 +12,3 @@ if errorlevel 1 exit 1
 
 ninja install
 if errorlevel 1 exit 1
-
-ninja test
```

Comment:
just curious why this was removed

Reply:
there are some test failing on windows and if you compare with build.sh on non-windows it is not even executed, so I have decided to be consistent and skip it on windows, too